relay-event-normalization/src/trimming.rs

```diff
@@ -280,37 +277,29 @@ fn trim_string(value: &mut String, meta: &mut Meta, max_chars: MaxChars) {
             let chunk_chars = chunk.count();
 
             // if the entire chunk fits, just put it in
-            if length + chunk_chars < soft_limit {
+            if length + chunk_chars < max_chars {
                 new_chunks.push(chunk);
                 length += chunk_chars;
                 continue;
             }
 
-            match chunk {
-                // if there is enough space for this chunk and the 3 character
-                // ellipsis marker we can push the remaining chunk
-                Chunk::Redaction { .. } => {
```

Comment:
I don't understand why this branch disappeared.

Reply:
It's back in [6c5e0d7](https://github.com/getsentry/relay/pull/2887/commits/6c5e0d7923dc75ff1d53046fd1f5adb9cd2839a0).